docs/make.jl

```diff
@@ -34,10 +34,6 @@ tutorials = [
         "Integrated soil+canopy modeling" => [
             "Coupled Canopy and Soil" => "integrated/soil_canopy_tutorial.jl",
         ],
-        "Bucket LSM" => [
```

Comment:
add back in once albedo file works?

Reply:
This is duplicated. There's another Bucket LSM above.